include/seastar/http/request.hh

```diff
@@ -121,6 +122,26 @@ struct request {
         return res->second;
     }
 
+    /**
+     * Search for the last path parameter of a given key
+     * @param key the path paramerter key
+     * @return the unescaped path parameter value, if it exists and can be path decoded successfully, otherwise it
+     *  returns an empty string
+     */
+    sstring get_path_param(const sstring& key) const {
```

Comment:
This method looks to better belong to `class parameters` rather than to request itself